src/components/modal/DonatePanel.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react'
+import { Col, Row } from 'reactstrap'
+import Modal from '../Modal'
+import { inject, observer } from 'mobx-react'
+import Gift from 'react-material-icon-svg/dist/GiftIcon'
+import i18nReact from 'i18n-react'
+
+interface DonatePanelInterface {
+  open: boolean
+  toggle: () => void
+}
+
+class DonatePanel extends React.Component<DonatePanelInterface> {
+
+  getLocaleId() {
+    return 'NL'
+  }
+
+  getBalance() {
+    return 4434
+  }
+
+  getPrice() {
+    return 333
+  }
+
+  render() {
+    const title: string = 'Donate'
+    return (
+      <Modal
+        {...this.props}
+        title={title}
+        className="donate-modal"
+      >
+        <p className="donate-description donate-description-larger">
+          At Verge we are always striving to achieve new heights and deliver products
+          or software implementations that are requested by the community.
+        </p>
+        <p  className="donate-description">
+          Listening to our community members and taking in feedback is not only
+          one of the core founding principles of Verge,
+          but it is inherent in our open-source nature.
+        </p>
```

Comment:
Those sentences should also be translated, let's wrap it for now and leave it english only. We will care about translations later on.